google/ggrc-core
# GGRC-4059 Cut off long GDrive folder title

# Issue description

Long title of Gdrive folder is not cut off and edit/delete folder controls are missing

# Steps to test the changes

1. Create control 
2. On the Control Info page attach folder with long title to control
3. Look at the 'edit'/'delete' folder controls: are absent

**Actual Result:** If user attaches a folder to control with long title:

- tile is not cut off;
- 'edit'/'delete' folder controls are absent.

**Expected Result:** Long title of Gdrive folder should be cut off.

**NOTE:** the same behavior for Audit folder, Workflow folder.

# Solution description

Fix css styles

# Sanity checklist




# PR Review checklist
